v2/clients/http/utils/common.go

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+import (
+	"context"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v2"
+
+	"github.com/google/uuid"
+)
+
+// FromContext allows for the retrieval of the specified key's value from the supplied Context.
+// If the value is not found, an empty string is returned.
+func FromContext(ctx context.Context, key string) string {
+	hdr, ok := ctx.Value(key).(string)
+	if !ok {
+		hdr = ""
+	}
+	return hdr
+}
+
+// Helper method to get the body from the response after making the request
+func getBody(resp *http.Response) ([]byte, error) {
```

Comment:
use errors.EdgeX to wrap all the standard error

Reply:
modified